catalog/measure/catalog-measure-api/src/main/java/ddf/measure/Distance.java

```diff
@@ -124,10 +125,52 @@ public double getAs(LinearUnit unitOfMeasure) {
     }
 
     /**
-     * The Enum LinearUnit for the distance in meters
+     * Enumeration for the linear units supported by DDF.
      */
     public enum LinearUnit {
-        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD
-    }
+        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD;
+
+        static Map<String, LinearUnit> stringsToValues = new HashMap<>();
```

Comment:
This map pattern is confusing. Why not just give your enum a constructor and override .name() or .getValue() or whatever? Technically it might be slightly more verbose, but I think it's much more readable and would get rid of the static block as well.